backend/app/Console/Commands/RenewEventSubscriptions.php

```
@@ -134,14 +201,44 @@ private function createOutlookEventSubscription(OutlookAccount $outlookAccount,
 
-            $outlookService->createEventSubscriptionByCalendar($outlookAccount, $display, $calendar->calendar_id);
+            // If subscription was created successfully, ensure retry tracking is reset
+            if ($eventSubscription) {
+                $eventSubscription->resetRetry();
+            }
         } catch (\Exception $e) {
-            $outlookAccount->update(['status' => AccountStatus::ERROR]);
-            $display->update(['status' => DisplayStatus::ERROR]);
+            // Try to create a placeholder subscription record for retry tracking
+            try {
+                $subscription = EventSubscription::create([
```

PHPStan (2.1.51)

[error] Call to an undefined static method App\Models\EventSubscription::create().


(staticMethod.notFound)